ding2.install

```diff
@@ -106,6 +106,13 @@ function ding2_install() {
     module_disable(array('overlay'));
   }
 
+  // Enable ding_user_form to allow provider login. It's enabled here as
+  //new/local sites should use "user form" and updated production should via
+  // update hooks turn on adgangsplatformen.
+  if (module_exists('ding_user_form')) {
+    module_enable(array('ding_user_form'));
+  }
```

Comment:
Why add this when [we allow the user to select which modules to enable during install - including `ding_user_form`](https://github.com/ding2/ding2/blob/5a7a2f9de2a3feb816e8c453a0c0874ce1cccdb8/ding2.profile#L378-L388)?

Reply:
`module_exists` will only return TRUE if the module is already enabled.

Reply:
Fixed